ProjectCoimbra.UWP/Project.Coimbra/Package.appxmanifest

```diff
@@ -9,7 +9,7 @@
 
   <Identity
     Name="62155AndreasBalzer.ProjectCoimbra-InternalTest"
-    Publisher="CN=DF151369-BF2A-4390-919A-40CE0C57CC96"
+    Publisher="CN=dimeehan"
```

Comment:
I guess you don't mean to change this...

Reply:
These changes got merged by mistake, I will revert.